pkg/gcp/validation/validation.go

```diff
@@ -17,6 +17,8 @@ import (
 const (
 	// DiskTypeScratch is the SCRATCH disk type
 	DiskTypeScratch = "SCRATCH"
+	// DiskTypePersistent is the PERSISTENT disk type
+	DiskTypePersistent = "PERSISTENT"
```

Comment:
minor nit: if you use these in the code, I would remove them from the validation package to the provider_spec package